app/controllers/registrations_controller.rb

```diff
@@ -26,6 +26,15 @@ def disable_two_factor
     end
   end
 
+  def tour
```

Comment:
You should check this error:
http://stackoverflow.com/questions/35071428/punditauthorizationnotperformederror
You must authorize your resource.